public/pages/CreateDetector/components/ConfigureAlerts/components/AlertCondition/AlertConditionPanel.tsx

```diff
@@ -85,57 +91,47 @@ export default class AlertConditionPanel extends Component<
 
   onTagsChange = (selectedOptions: EuiComboBoxOptionOption<string>[]) => {
     const tags = selectedOptions.map((tag) => tag.label);
-    const {
-      alertCondition,
-      changeDetector,
-      detector,
-      detector: { triggers },
-      indexNum,
-    } = this.props;
-    triggers.splice(indexNum, 1, { ...alertCondition, tags: tags });
-    changeDetector({ ...detector, triggers: triggers });
+    this.updateTrigger({ tags });
   };
 
   onNotificationChannelsChange = (selectedOptions: EuiComboBoxOptionOption<string>[]) => {
     // const channelIds = selectedOptions.map((channel) => channel.label);
     const {
```

Comment:
nit: done with commented code?

Reply:
No, will be using it when integrating with notifications API.